Loop/Views/BolusEntryView.swift

```diff
@@ -256,11 +256,7 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
             HStack(alignment: .firstTextBaseline) {
                 Text(recommendedBolusString)
                     .font(.title)
-                    .foregroundColor(viewModel.enteredBolus.doubleValue(for: .internationalUnit()) == 0 && viewModel.isBolusRecommended ? .accentColor : Color(.label))
-                    .onTapGesture {
-                        self.viewModel.acceptRecommendedBolus()
-                    }
-
```

Comment:
Remove ability to accept recommended bolus by tapping on text (since now it's auto-accepted)